server/src/test/java/org/spine3/server/aggregate/AggregatePartShould.java

```diff
@@ -75,6 +101,20 @@ public void create_aggregate_part_entity() throws NoSuchMethodException {
         assertNotNull(aggregatePart);
     }
 
+    @Test
+    public void return_aggregate_part_state_by_class() {
+        taskRepository.store(taskPart);
+        final Task task = taskDescriptionPart.getPartState(Task.class);
+        assertEquals(TASK_DESCRIPTION, task.getDescription());
+    }
+
+    @Test(expected = NullPointerException.class)
+    public void throw_exception_when_trying_to_getpartstate_from_null_aggregate_part() {
+        taskRepository.store(taskPart);
+        final Task task = taskDescriptionPart.getPartState(null);
+        assertEquals(TASK_DESCRIPTION, task.getDescription());
```

Comment:
Seems like this line is never reached since the `expected` Exception is thrown on the line above.
Please delete this line and check similar cases.